packages/knip/fixtures/plugins/unocss/unocss.config.ts

```diff
@@ -0,0 +1,6 @@
+// uno.config.ts
+import { defineConfig } from 'unocss'
+
+export default defineConfig({
+  // ...UnoCSS options
```

Comment:
I think it would be good to use a more comprehensive config file here to make it more real-world.